src/kit/LogViewer/LogViewer.tsx

```diff
@@ -553,50 +529,76 @@ const LogViewer: React.FC<Props> = ({
   );
 
   return (
-    <Section
-      bodyNoPadding
-      bodyScroll
-      divider
-      maxHeight
-      options={logViewerOptions}
-      title={props.title}>
-      <Spinner center spinning={isFetching} tip={logs.length === 0 ? 'No logs to show.' : ''}>
-        <div className={css.base} ref={baseRef}>
-          <div className={css.container}>
-            <div className={css.logs} ref={refCallback}>
-              <VariableSizeList
-                height={pageSize.height - 250}
-                itemCount={logs.length}
-                itemData={logs}
-                itemSize={getItemHeight}
-                ref={listRef}
-                width="100%"
-                onItemsRendered={handleItemsRendered}
-                onScroll={handleScroll}>
-                {LogViewerRow}
-              </VariableSizeList>
-            </div>
-          </div>
-          <div className={css.buttons} style={{ display: showButtons ? 'flex' : 'none' }}>
-            <Button
-              aria-label={ARIA_LABEL_SCROLL_TO_OLDEST}
-              icon={<Icon name="arrow-up" showTooltip title={ARIA_LABEL_SCROLL_TO_OLDEST} />}
-              onClick={handleScrollToOldest}
+    <Section>
+      <Column>
```

Comment:
fix: this isn't what I meant by using Columns. It should look like this:
```
<Columns>
  <Column>
    <h3>{props.title}</h3>
  </Column>
  <Column align="right">
    <Space>
      {... options ...}
    </Space>
  </Column>
</Columns>
```
and the sectionBody shouldn't be in a column